### PR TITLE
cleanup history prior to pull request, intro only

### PR DESCRIPTION
**Pull request by rogermeadows**
This is first draft of a new document that describes how to clean up history in your working directory prior to creating a pull request. It contains only the introduction section for now. Once the introduction is reviewed and approved I will add detailed instructions and put a link to this new page in the top level readme file.

### CODE REVIEW

File: cleanup_b4_pull_request.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Cleaning up history prior to creating a pull request
+The idea of cleaning up your branch's history before creating a pull request can be a controversial topic.
+Some project managers require some level of cleanup and some do not. Sometimes it is up to the developer to 
+decide how much (if any) cleanup is done before creating a pull request. Here we will discuss some of 
+the factors to consider when deciding how much (if any) clean up you may want to do before creating a
+pull request.
+We will also explain how to perform some of the cleanup options.
+This discussion assumes we are using in a fork/clone development workflow.
```
Comment (SunyataZero) [inline cleanup_b4_pull_request.md:8]: > fork/clone development workflow

Can we link to this file? https://github.com/mindfulness-at-the-computer/git-info/blob/master/fork-clone-workflow.md

**Review by SunyataZero — APPROVED** [review]
Very interesting for me to read, there's a lot of stuff here that are new to me, thanks for sharing all this info!

File: cleanup_b4_pull_request.md (hunk 1 above)
Comment (rogermeadows) [reply cleanup_b4_pull_request.md:8]: Yes, but I was planning on filling in the instructions part before linking it in.
Are OK with merging this version (not linked in yet) while I work on the rest of the document?

Comment (SunyataZero) [reply cleanup_b4_pull_request.md:8]: Alright, sounds good!

**Pull request merged**